<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelB">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditA">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Вычислить</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculate()</slot>
 </slots>
</ui>
